### PR TITLE
Fix the Autoroute part UI

### PR DESCRIPTION
**Pull request by MikeAlhayek**
Updating the setting UI for the Autoroute settings

## After the change screenshots
image

image

## Before the change screenshots

image

### CODE REVIEW

**Comment by hishamco** [comment]
Do we really needs all these changes for adding a space ;)

**Comment by MikeAlhayek** [comment]
It's adding a space and also allow the show/hide as the code was intended.

File: src/OrchardCore.Modules/OrchardCore.Autoroute/Views/AutoroutePartSettings.Edit.cshtml [hunk 1]
```diff
@@ -99,5 +100,5 @@
             checked ? $(this).show() : $(this).hide();
         });
     });
-    //]]>
+                                                            //]]>
```
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.Autoroute/Views/AutoroutePartSettings.Edit.cshtml:103]:
```suggestion
    //]]>
```

**Review by Skrypt — APPROVED** [review]

**Comment by hishamco** [comment]
> It's adding a space and also allow the show/hide as the code was intended.

So, I will leave this to Jasmin

**Pull request merged**